broker-cli/commands/wallet.js

```diff
@@ -521,7 +546,8 @@ module.exports = (program) => {
     .command(`wallet ${SUPPORTED_COMMANDS.NETWORK_STATUS}`, 'Payment Channel Network status for trading in different markets')
     .option('--market [marketName]', 'Relevant market name', validations.isMarketName)
     .command(`wallet ${SUPPORTED_COMMANDS.RELEASE}`, 'Closes channels open on the specified market')
-    .option('--market [marketName]', 'Relevant market name', validations.isMarketName)
+    .option('--market <marketName>', 'Relevant market name', validations.isMarketName, null, true)
+    .option('--force [force]', 'Relevant market name', null, false)
```

Comment:
I don't think force takes a parameter, so it should just be `--force`, not `--force [force]`, right? Unless I'm misunderstanding the caporal syntax

Reply:
nope! you are all good. thanks for the catch